.github/workflows/other.yml

```diff
@@ -0,0 +1,138 @@
+name: Build and test (other)
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # run daily, this refreshes the cache
+    - cron: "13 2 * * *"
+
+concurrency: # On new push, cancel old workflows from the same PR, branch or tag:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  python-test:
+    name: Python tests
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["2.7", "3.11"]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # To check which files changed: origin/master..HEAD
+      - uses: LizardByte/setup-python-action@master
+        with:
+          python-version: ${{matrix.python-version}}
+
+      - uses: actions/cache@v4
+        name: Setup cache for running pre-commit fast
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
+
+      - run: echo "::add-matcher::.github/workflows/python-warning-matcher.json"
+        name: "Setup GitHub for reporting Python warnings as annotations in pull request code review"
+
+      - uses: pre-commit/action@v3.0.1
+        name: Run pre-commit checks (no spaces at end of lines, etc)
+        if: ${{ matrix.python-version != '2.7' }}
+        with:
+          extra_args: --all-files --verbose --hook-stage commit
+        env:
+          SKIP: no-commit-to-branch
+
+      - name: Install dependencies only needed for python 2
+        if: ${{ matrix.python-version == '2.7' }}
+        run: pip install enum
+
+      - name: Install dependencies only needed for python 3
+        if: ${{ matrix.python-version != '2.7' }}
+        run: pip install opentelemetry-api opentelemetry-exporter-zipkin-json opentelemetry-sdk pandas pytype toml wrapt pyudev
+
+      - name: Install common dependencies for Python ${{matrix.python-version}}
+        run: pip install future mock pytest-coverage pytest-mock
+
+      - name: Run Pytest for python 2 and get code coverage for Codecov
+        if: ${{ matrix.python-version == '2.7' }}
+        run: >
+          pytest
+          --cov=scripts --cov=ocaml/xcp-rrdd
```

Comment:
line 63:
This line has been modified in the feature branch to be:
`--cov=ocaml/xcp-rrdd`

Reply:
That may be, but as long as there are Python scripts in the `scripts` directory, we should not remove testing them.